share/spice/yoda_speak/yoda_speak.css

```diff
@@ -0,0 +1,15 @@
+.zci__body h3 {
+     font-size: 2.0em;
+     font-weight: 400;
+     color: #3f3f3f;
```

Comment:
We should assign a color via one of the classes specified in https://duckduckgo.com/styleguide.

Same with below.